openstack/compute/v2/extensions/instanceactions/results.go

```diff
@@ -0,0 +1,87 @@
+package instanceactions
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// InstanceAction represents an instance action.
+type InstanceAction struct {
+	Action       string    `json:"action"`
+	InstanceUUID string    `json:"instance_uuid"`
+	Message      *string   `json:"message"`
```

Comment:
Having this as `*string` will cause the value to be `nil` if there is no message. Is there a reason to do this instead of having an empty value be `""`? If there is, I'm OK with keeping this as `*string`.

Reply:
It permits to say to the dev that the field be empty.
If it's a pointer is easier for the dev to konw the field is optional and must be checked before used.